### DISCUSSION TITLE
ISIS 8.2.0 Release Candidate 2 is now available

### DISCUSSION THREAD

**Discussion by chkim-usgs** [DOI-USGS/ISIS3#5532] [General]
## How to install or update to 8.2.0_RC2

Installation instructions of ISIS can be found in the README on our [github page ](https://github.com/DOI-USGS/ISIS3). 

If you already have a version of ISIS 4.0.0 or later installed in an anaconda environment, you can update to 8.2.0_RC2 by activating your existing isis conda environment and running `conda update -c "usgs-astrogeology/label/RC" isis=8.2.0_RC2`.

### How to get access to 8.2.0_RC2 at the ASC 

The new process proposed in the internal [RFC](https://astrodiscuss.usgs.gov/t/internal-rfc-distribution-of-isis3-at-asc/52/26) is now in full effect. Please review the process of using anaconda environments to activate isis [here](https://astrodiscuss.usgs.gov/t/using-the-asc-conda-environment-for-isisx-y-z/106). 

Once a version of conda is active, run the command: `conda activate isis8.2.0-RC2` to use this newest version of ISIS. 

### Changed
- Changed `qwt` dependency version to 6.2.0 or below [#5498](https://github.com/DOI-USGS/ISIS3/issues/5498)
- Pinned `suitesparse` dependency version to maximum not including 7.7.0 [#5496](https://github.com/DOI-USGS/ISIS3/issues/5496)

### Fixed
- Fixed a bug in QVIEW's Stretch tool where the default min/max type was not an available option [#5289](https://github.com/DOI-USGS/ISIS3/issues/5289)
- Fixed a bug in QVIEW where images would double load if loaded from the commandline [#5505](https://github.com/DOI-USGS/ISIS3/pull/5505)
- Fixed <i>noproj</i> bug where segfault occurs if match cube is provided [#5507](https://github.com/DOI-USGS/ISIS3/pull/5507)
- Fixed external dependencies causing double free and other memory access errors

## Notes 

The following operating systems are supported for this release: 

* Ubuntu 18.04 
* macOS Mohave 11.6

(Other Linux/macOS variants may be able to run this release, but are not officially supported.) 

If you find a problem with this release, please create an issue on our [github issues page](https://github.com/DOI-USGS/ISIS3/issues/new/choose/)

**Comment by acpaquette** [comment]
For anyone interested, the 8.2.0_RC2 should now be fixed and testable!
